modules/gapi/include/opencv2/gapi/gcall.hpp

```diff
@@ -56,11 +56,14 @@ class GAPI_EXPORTS GCall final
     Priv& priv();
     const Priv& priv() const;
 
-protected:
-    std::shared_ptr<Priv> m_priv;
+    GKernel& kernel();
```

Comment:
Why? Is kernel modifiable?

Reply:
Because `GInferOutput::at` expands `GKernel::outShapes`

Reply:
Please comment this if you haven't yet

Reply:
Added